src/loan_form/loan_entry/MonthlyPayment.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import LoanEntryField from './LoanEntryField'
+import Enum from './enum'
+
+const { LOAN_ENTRY_COPY: { MONTHLY_PAYMENT: { label, placeholder } } } = Enum
+
+export default function MonthlyPayment({ amount, onChange }) {
+  return (
+    <LoanEntryField
+      fieldClass='monthly-payment'
+      name='monthly-payment'
```

Comment:
Is there a reason this and Title.jsx have name attributes but not the other fields? They don't seem like they're necessary anymore